arches_controlled_lists/utils/skos.py

```diff
@@ -0,0 +1,204 @@
+import uuid
+import re
+from itertools import chain
+from django.db.models import Q
+from django.db import transaction
+from rdflib import Namespace, RDF
+from rdflib.namespace import SKOS, DCTERMS
+from rdflib.graph import Graph
+from arches.app.models import models
+from arches.app.models.system_settings import settings
+from arches.app.utils.skos import SKOSReader
+from arches_controlled_lists.models import List, ListItem, ListItemValue
+
+# define the ARCHES namespace
+ARCHES = Namespace(settings.ARCHES_NAMESPACE_FOR_DATA_EXPORT)
+
+
+class SKOSReader(SKOSReader):
+    def __init__(self):
+        super().__init__()
+        self.lists = {}
+        self.list_items = {}
+        self.list_item_values = []
+
+    """
+    This class extends the SKOSReader to provide additional functionality
+    specific to the Arches controlled lists application.
+    """
+
+    def save_controlled_lists_from_skos(
+        self,
+        graph,
+        # overwrite_options="overwrite",
```

Comment:
Might be worth chatting about your ideas here -- if we're not supporting it then we might want to gracefully skip conflicts instead of letting bulk_create fail.